src/main/test/com/social/VkPostTest.java

```diff
@@ -0,0 +1,33 @@
+package com.social;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import com.social.interfaces.ISocialNetworkService;
+import com.social.models.Post;
+import com.social.models.SocialNetwork;
+import com.social.models.SocialNetworkType;
+import com.social.services.SocialNetworkFactory;
+import com.social.utils.PropertyService;
+
+public class VkPostTest {
+	private String vkLogin = PropertyService.getValueProperties("vkLogin");
```

Comment:
Why you have moved those two fields here? Do you use them in the other tests or in the other places?